Atomic/ps.py

```diff
@@ -24,6 +30,8 @@ def cli(subparser):
                      help=_("Don't truncate output"))
     pss.add_argument("-q", "--quiet", action='store_true', dest="quiet", default=False,
                      help=_("Only display container IDs"))
+    pss.add_argument("--fstrim", action='store_true', dest="fstrim", default=False,
```

Comment:
this should be added to the documentation and bash autocompletion as well.  There is a `test_ps.sh` file that you could use to add a test as well.